Better Package Style layout (#71)
pyansys/dev-guide: doc/source/overview/administration.rst

Each repository is expected to follow this minimum set of standards:

> - Minimum code standards following PEP8. See :ref:`best_practices`.

Reviewer: ```suggestion
- PEP8's code standards. See :ref:`best_practices`.
```

- CI/CD using GitHub actions or Azure DevOps to enforce coding standards.

> - Publicly hosted documentation detailing API with examples. See

Reviewer: ```suggestion
- Publicly hosted documentation describing the API and providing examples. See
```
